moviebot/nlu/annotation/semantic_annotation.py

```diff
@@ -50,28 +54,31 @@ def __init__(
         if annotation_type == AnnotationType.NAMED_ENTITY:
             self.entity_type = entity_type
 
-    def get_type(self) -> Text:
-        entity_type = " - " + self.entity_type.name if self.entity_type else ""
-        return (self.annotation_type.name + entity_type).lower()
+    def get_type(self) -> str:
+        """Returns semantic annotation type as string."""
+        entity_type_str = (
+            f" - {self.entity_type.name}" if self.entity_type else ""
+        )
+        return (self.annotation_type.name + entity_type_str).lower()
 
     @classmethod
     def from_span(
         cls,
         span: Span,
         annotation_type: AnnotationType,
         entity_type: Optional[EntityType] = None,
-    ):
-        """Create semantic annotation from already existing span or token. This
+    ) -> "SemanticAnnotation":
```

Comment:
Why is SemanticAnnotation in ""?

Reply:
Because SemanticAnnotation does not exist yet. It is a reference to a class where the typed method is defined.